README.md

```diff
@@ -40,3 +40,15 @@ The research team are supported by the Imperial College London
 [Research Software
 Engineering](https://www.imperial.ac.uk/admin-services/ict/self-service/research-support/rcs/service-offering/research-software-engineering/)
 team.
+
+## Example usage
+
+An example configuration for the Virtual Rainforest model is provided in
+`data/example_config.toml`. A simulation using this configuration can be run through the
+command line by calling:
+
+```shell
+poetry shell
+poetry install
+vr_run data/example_config.toml
+```
```

Comment:
I don't think this is the right place for this. The README is very simple and this example is really out of context and hard to follow here. I think this needs to be in the `docs` and be the beginnings of a basic how to page.

Also the `poetry` installation is very much the developer mode. Most end users will `pip` it - and then there is the problem that the path of the config file will be within the local python library and not the working directory. There must be a canonical way to simplify user access to provided example files!

Reply:
If you do move this, a good time to add that RTD link 😄

Reply:
And promote @TaranRallings !

Reply:
Ahh yeah fair enough, I will move it to a basic how to page

Reply:
And I think `install` should be before `shell` if you do hang on to to the `poetry` installation for the moment.

Reply:
When it is `pip` installed, _all_ of the package is (normally) installed to one of the `site-packages` locations (user or system). The `vr_run` command is installed so that it can be found from `PATH`, but the path to any specific file in the package is going to be obscure. I think the solution is that we want a `vr install_demo` command that can retrieve the config and some accompanying data files into a given location. That can programatically access the file locations and then additionally make sure that users play around with _copies_ of the files rather than corrupting the package data.